endpoint/endpointComp/endpoint.c

```diff
@@ -25,8 +25,8 @@
   "999999B"
 #define TEST_DEVICE_ID "470010171566423"
 
-const uint8_t test_key[32] = {82,  142, 184, 64,  74, 105, 126, 65,  154, 116, 14,  193, 208, 41,  8,  115,
-                              158, 252, 228, 160, 79, 5,   167, 185, 13,  159, 135, 113, 49,  209, 58, 68};
+const uint8_t test_private_key[32] = {82,  142, 184, 64,  74, 105, 126, 65,  154, 116, 14,  193, 208, 41,  8,  115,
```

Comment:
Change the constant value to AES_CBC_KEY_SIZE.

Reply:
Done.